Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by brainfood1183 (github), altered by discord: mrgreen06 (358779039810322433)",
```

Comment:
Add a link to a commit with the sprite (as it's currently used, just the current commit should be fine).